templates/bonsai/Cargo.toml

```diff
@@ -8,9 +8,9 @@ members = [
 [workspace.dependencies]
 hello-bonsai-methods = { path = "methods" }
 hello-bonsai-contracts = { path = "contracts" }
-risc0-zkp = { version = "0.13" }
-risc0-zkvm = { version = "0.13" }
-risc0-build = { version = "0.13" }
+risc0-zkp = { path = "../../risc0/zkp" }
+risc0-zkvm = { path = "../../risc0/zkvm" }
+risc0-build = { path = "../../risc0/build" }
```

Comment:
I think we might want both, which will:
1. When used locally, it uses the `path`.
2. When the crate is published, it uses the `version`.

Or perhaps we can arrange for the `cargo risc0` tool to tweak this when it deploys the template.

Reply:
Are you saying to have `risc0-zkp = { path = "../../risc0/zkp", version = "0.13" }`? Would that work?

Another option I thought about would be to have this handled by a patch statement down below, making it easy to copy the template out and simply drop the patch statements, but that feels a bit error prone, and we can figure out the right way to do this when we write the `cargo risc0` template generator.

Reply:
Yes, you can specify both `path` and `version`, but I'm not entirely sure if that's actually what we want here.

Agreed that maybe best place to solve this is with the generator tool.